packages/neon-ledger/src/ErrorCode.ts

```diff
@@ -22,20 +37,21 @@ export interface TransportStatusError extends Error {
 export function evalTransportError(err: Error): Error {
   const transportErr = err as TransportStatusError;
   switch (transportErr.statusCode) {
-    case ErrorCode.APP_CLOSED:

```

Comment:
Have you checked if this code still exists? I believe this code is external to the NEO app and it might still work. It is used mainly at the login screen when we detect a ledger but without the app open. Might be useful to still have this.

Reply:
Ah I didn't see that code anywhere in Ledger's new boilerplate app (what I based the new NEO app on). It seems to have changed from `0x6e00` to `0x6e01`. I moved it back in.